### PR TITLE
malus: correct version

### PR DESCRIPTION
**Pull request by i0ntempest**
**Important:** *Do not tick a checkbox if you haven’t performed its action.* Honesty is indispensable for a smooth review process.

After making all changes to a cask, verify:

- [x] `brew cask audit --download {{cask_file}}` is error-free.
- [x] `brew cask style --fix {{cask_file}}` reports no offenses.
- [x] There are no [open pull requests](https://github.com/Homebrew/homebrew-cask/pulls) for the same update.
- [x] The submission is for [a stable version](https://github.com/Homebrew/homebrew-cask/blob/master/doc/development/adding_a_cask.md#stable-versions) or [documented exception](https://github.com/Homebrew/homebrew-cask/blob/master/doc/development/adding_a_cask.md#but-there-is-no-stable-version).

Additionally, **if adding a new cask**:

- [ ] Named the cask according to the [token reference](https://github.com/Homebrew/homebrew-cask/blob/master/doc/cask_language_reference/token_reference.md).
- [ ] `brew cask audit --new-cask {{cask_file}}` worked successfully.
- [ ] `brew cask install {{cask_file}}` worked successfully.
- [ ] `brew cask uninstall {{cask_file}}` worked successfully.
- [ ] Checked the cask was not [already refused](https://github.com/Homebrew/homebrew-cask/search?q=is%3Aclosed&type=Issues).
- [ ] Checked the cask is submitted to [the correct repo](https://github.com/Homebrew/homebrew-cask/blob/master/doc/development/adding_a_cask.md#finding-a-home-for-your-cask).

### CODE REVIEW

**Comment by i0ntempest** [comment]
Upstream got the file name version wrong again. I think we should keep two versions separated by comma from now on for this cask even if the versions do match just in case.

**Comment by i0ntempest** [comment]
Or we can use the upper level URL https://getmalus.com/r/mac which redirects to the file to avoid this hassle, optionally with `latest` and `no_check`

File: Casks/malus.rb [hunk 1]
```diff
@@ -1,5 +1,5 @@
 cask "malus" do
-  version "1.5.2"
+  version "1.5.1,152"
```
Comment (vitorgalvao) [inline Casks/malus.rb:2]: This way is simpler and less confusing (`152` looks more like a build number).

**Review by vitorgalvao — DISMISSED** [review]

File: Casks/malus.rb (hunk 1 above)
Comment (i0ntempest) [reply Casks/malus.rb:2]: Thanks for this, but in the other line you changed I think u deleted the wrong part in the expression (you prolly meant to delete `no_dots`)

**Review by vitorgalvao — APPROVED** [review]

**Pull request merged**